openedx/core/djangoapps/credit/tasks.py

```diff
@@ -208,17 +211,19 @@ def _get_proctoring_requirements(course_key):
     # process
     from edx_proctoring.api import get_all_exams_for_course
 
-    requirements = [
-        {
-            'namespace': 'proctored_exam',
-            'name': exam['content_id'],
-            'display_name': exam['exam_name'],
-            'criteria': {},
-        }
-        for exam in get_all_exams_for_course(unicode(course_key))
-        # practice exams do not count towards eligibility
-        if exam['is_proctored'] and exam['is_active'] and not exam['is_practice_exam']
-    ]
+    requirements = []
+    for exam in get_all_exams_for_course(unicode(course_key)):
+        if exam['is_proctored'] and exam['is_active'] and not exam['is_practice_exam']:
+            usage_key = UsageKey.from_string(exam['content_id'])
+            sequential = modulestore().get_item(usage_key)
```

Comment:
Its safe to assume that this is `sequential` but I am more comfortable using just `block` as name since in future `proctored exam` may not be directly connected to a `sequential` module.